src/components/ClusterOverview/TopConsumers/strings.js

```diff
@@ -0,0 +1,17 @@
+export const BY_CPU = 'By CPU';
+export const CPU_DESC = 'CPU Time';
```

Comment:
I think it would make more sense to make a mapping of that

Reply:
mapping is done in https://github.com/kubevirt/web-ui-components/pull/289/files#diff-2d8caae240c01140ce53da1f24c6cdb4R37  and https://github.com/kubevirt/web-ui-components/pull/289/files#diff-2d8caae240c01140ce53da1f24c6cdb4R39 or do you mean some other mapping ? Would it bring any additional value ?